src/index.html

```diff
@@ -15,8 +15,96 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
```

Comment:
Consider adding a more descriptive `alt` attribute for the image. The `alt` attribute should describe the image content, not just say 'logo'. For example, 'Moyo company logo'.

Reply:
> Consider adding a more descriptive `alt` attribute for the image. The `alt` attribute should describe the image content, not just say 'logo'. For example, 'Moyo company logo'.

Good point
<img width="467" alt="image" src="https://github.com/user-attachments/assets/39a18286-70c1-43ba-b21d-15c4a65f0077">